site/docs/references/mixin-syntax.md

```diff
@@ -1,124 +1,253 @@
 ---
-id: references/mixin-syntax
+id: references/mixins-syntax
 title: Mixins
 layout: docs
 ---
 
-**Stylable** mixins allow you to reuse complex styles and CSS behaviors, and apply them to any ruleset. 
 
-Some use cases where you can use mixins:
+**Stylable** mixins enable you to reuse complex styles and CSS behaviors, and apply them to any ruleset during build time. 
+
+Here are some examples of when you can use mixins:
+
 * Presets/Variants - create reusable pieces of styling CSS
 * Layouts - easily describe complex layouts
 * Effects - easily describe complex effects
-* Macros - use JS to define the CSS macros you need
+* Macros - use code to define the CSS macros you need
+
+Mixins can be applied using either CSS or JavaScript, and can recive parameters.
```

Comment:
I think this should be mixed-in with the first sentence:

**Stylable** mixins enable you to reuse complex styles and CSS behaviors, defined in CSS or Javascript, and apply them to any ruleset during build time.